src/main/scala/ru/dsr/bigdata/Parameters.scala

```diff
@@ -0,0 +1,27 @@
+package ru.dsr.bigdata
+
+import com.typesafe.config.Config
+
+
+object Parameters {
+  def getInstanceOf(conf: Config) = new Parameters(conf)
+}
```

Comment:
Create `AppConfig` object instead of `Parameters` class. Something like that: 
```
object AppConfig {
  val conf = ConfigFactory.load.getConfig("app")
  val data: Config = conf.getConfig("data")
  val load_from: String = data.getString("load_from")
  val save_to: String = data.getString("save_to")
...
```